backend/tests/test_SRC_1.py

```diff
@@ -0,0 +1,76 @@
+import base64
+import unittest
+import json
+
+from model.books import BooksModel
+from model.users import UsersModel
+from tests.base_test import BaseTest
+from model.transactions import TransactionsModel
+
+
+class UnitTestOfUS(BaseTest):
+
+# TEST TASK 1
+    def test_get_search_by_isbn(self):
+        with self.app.app_context():
+            book = BooksModel(1, 1, 1, "test")
+            book.save_to_db()
+            book = BooksModel(2, 1, 1, "as")
+            book.save_to_db()
+            args = {
+                "isbn": 1,
+                'autor': "patata"
+            }
+            res = self.client.get('/search', data=args)
+            self.assertEqual(200, res.status_code)
+            self.assertEqual(len(json.loads(res.data)["books"]), 1)
+            list_books = list(map(lambda u: u.json(), BooksModel.query.filter_by(isbn=args['isbn']).all()))
```

Comment:
Esta be pero hagues sigut mes facil fer: 
```python
list_books = list(BooksModel.find_by_isbn(args['isbn'])
```

Reply:
no funciona, ho deixo com abans